libnopegl/src/scene.c

```diff
@@ -199,6 +203,37 @@ static int build_nodes_set(struct ngl_scene *s)
     return ret;
 }
 
+static int track_files(struct ngl_scene *s)
+{
+    ngli_darray_init(&s->files, sizeof(char *), 0);
+    ngli_darray_init(&s->files_par, sizeof(uint8_t *), 0);
```

Comment:
`void *`

Reply:
We use `uint8_t *` for the params agnostic pointers at the moment. I don't mind another type but I'd rather keep it consistent with the rest of the codebase.